mclagsyncd/mclag.h

```diff
@@ -77,8 +99,17 @@ typedef enum mclag_msg_type_e_ {
     MCLAG_MSG_TYPE_FLUSH_FDB = 3,
     MCLAG_MSG_TYPE_SET_INTF_MAC = 4,
     MCLAG_MSG_TYPE_SET_FDB = 5,
-    MCLAG_MSG_TYPE_FLUSH_FDB_BY_PORT = 6,
-    MCLAG_MSG_TYPE_GET_FDB_CHANGES = 20
```

Comment:
I this "MCLAG_MSG_TYPE_GET_FDB_CHANGES" not being used before?  I did not see this on the set of your new changes.  I do see MCLAG_MSG_TYPE_FLUSH_FDB_BY_PORT got changed from 6 to 20.

Reply:
GET_FDB_CHANGES and FLUSH_FDB_BY_PORT is no more used. removed it. have checked in new changes aligning https://github.com/Azure/sonic-buildimage/pull/4819 iccpd/include/msg_format.h